apps/functions-subscription/api/internal.yaml

```diff
@@ -0,0 +1,128 @@
+openapi: 3.0.1
+info:
+  version: 0.0.0
+  title: Trial System API
+  description: |
+    Documentation of the Trial System Subscription function.
+servers:
+  - url: https://ts-p-itn-subscription-fn-01.azurewebsites.net
+security:
+  - ApiKeyAuth: []
+paths:
+  /info:
+    get:
+      operationId: info
+      responses:
+        '200':
+          $ref: '#/components/responses/200Info'
+        '404':
+          $ref: '#/components/responses/404NotFound'
+        '500':
+          $ref: '#/components/responses/500InternalServerError'
+  /subscriptions/{trialId}:
+    get:
+      operationId: getSubscription
+      summary: Get subscription detail
+      description: >-
+        Returns the details about the subscription of the specified user for
+        the specified trial.
+      parameters:
+        - $ref: '#/components/parameters/pathTrialId'
+        - $ref: '#/components/parameters/headerUserId'
```

Comment:
**note**: I've put the `userId` in the header because it seems to me the better place, but we can move it somewhere else

Reply:
It seems reasonable. We must check on the backend side how this affect client generation (It should work fine)